tests/ptable/test_ptable_matplotlib.py

```diff
@@ -154,155 +154,88 @@ def steel_elem_counts(steel_formulas: pd.Series[Composition]) -> pd.Series[int]:
+        assert len(fig.axes) == 127 if hide_f_block is True else 181, len(fig.axes)
 
-    # cbar_fmt as string
-    ax = ptable_heatmap(glass_elem_counts, cbar_fmt=".3f")
-    cbar_labels = [label.get_text() for label in ax.child_axes[0].get_xticklabels()]
-    assert cbar_labels[:2] == ["0.000", "50.000"]
+    @pytest.mark.parametrize("log", [False, True])
+    def test_log_scale(self, log: bool) -> None:
+        fig = ptable_heatmap(df_ptable.atomic_mass, log=log)
+        assert isinstance(fig, plt.Figure)
+        assert len(fig.axes) == 181
 
-    # cbar_fmt as function
-    ax = ptable_heatmap(glass_elem_counts, fmt=si_fmt)
-    ax = ptable_heatmap(
-        glass_elem_counts, fmt=lambda x: f"{x:.0f}", cbar_fmt=si_fmt_int
+    @pytest.mark.parametrize(
+        "values_show_mode", ["percent", "fraction", "value", "off"]
     )
-    ax = ptable_heatmap(glass_elem_counts, cbar_fmt=lambda x, _: f"{x:.3f} kg")
-
-    ax = ptable_heatmap(glass_elem_counts, heat_mode="percent", cbar_fmt=".3%")
-    cbar_ax = ax.child_axes[0]
-    cbar_1st_label = cbar_ax.get_xticklabels()[0].get_text()
-    assert cbar_1st_label == "0.000%"
-    cbar_title = cbar_ax.title
-    assert str(cbar_title) == "Text(0.5, 1.0, 'Element Count')"
-    # check colorbar title font color is black and hence visible on white background
-    assert cbar_title.get_color() == "black"
-
-    # tile_size
-    ptable_heatmap(df_ptable.atomic_mass, tile_size=1)
-    ptable_heatmap(df_ptable.atomic_mass, tile_size=(0.9, 1))
-
-    # bad colorscale should raise ValueError
-    bad_name = "bad color scale"
-    with pytest.raises(
-        ValueError,
-        match=f"{bad_name!r} is not a valid value for name; supported values are "
-        "'Accent', 'Accent_r'",
-    ):
-        ptable_heatmap(glass_formulas, colorscale=bad_name)
-
-    # test text_style
-    ptable_heatmap(glass_formulas, text_style=dict(color="red", fontsize=12))
-
-    # test show_scale (with heat_mode)
-    ptable_heatmap(glass_formulas, heat_mode="percent", show_scale=False)
-
-
-def test_ptable_heatmap_text_color_consistency(glass_formulas: list[str]) -> None:
-    ax = ptable_heatmap(glass_formulas)
-    check_text_color_consistency(ax)
-
-    ax = ptable_heatmap(glass_formulas, log=True)
-    check_text_color_consistency(ax)
-
-    ax = ptable_heatmap(glass_formulas, text_color=("red", "blue"))
-    check_text_color_consistency(ax)
-
-    ax = ptable_heatmap(glass_formulas, heat_mode="percent")
-    check_text_color_consistency(ax)
-
-    ax = ptable_heatmap(glass_formulas, show_values=False)
-    check_text_color_consistency(ax)
-
+    def test_values_show_mode(
+        self, values_show_mode: Literal["percent", "fraction", "value", "off"]
+    ) -> None:
+        fig = ptable_heatmap(df_ptable.atomic_mass, values_show_mode=values_show_mode)
+        assert isinstance(fig, plt.Figure)
+        assert len(fig.axes) == 181
 
-def check_text_color_consistency(ax: plt.Axes) -> None:
```

Comment:
I would remove this for now because with current implementation symbols and values would take color from the same mapping, so would expect them to be always consistent. 

Mark this because it's always better to have redundancy in tests, but I currently cannot think of a proper way to get the colors of values/symbols.